components/dash-core-components/src/components/Checklist.react.js

```diff
@@ -33,68 +33,87 @@ export default class Checklist extends Component {
                 style={style}
                 className={className}
             >
-                {options.map(option => (
-                    <label
-                        key={option.value}
-                        style={labelStyle}
-                        className={labelClassName}
-                    >
-                        <input
-                            checked={includes(option.value, value)}
-                            className={inputClassName}
-                            disabled={Boolean(option.disabled)}
-                            style={inputStyle}
-                            type="checkbox"
-                            onChange={() => {
-                                let newValue;
-                                if (includes(option.value, value)) {
-                                    newValue = without([option.value], value);
-                                } else {
-                                    newValue = append(option.value, value);
-                                }
-                                setProps({value: newValue});
-                            }}
-                        />
-                        {option.label}
-                    </label>
-                ))}
+                {options.map(option => {
+                    option =
+                        typeof option === 'string'
```

Comment:
I noticed that other parts of the code use `type()` method from the `ramda` package for type checking, actually, I think it wouldn't add a lot, maybe just a bit more of consistency.

Reply:
I agree with you and better to have more consistency. 7497eb218b3545633b11fbeeba5a56bc7f49880b